std_msgs/Header header

uint32 agent_id
string agent_role
string agent_state
geometry_msgs/Point position
geometry_msgs/Vector3 velocity
float32 battery_level
string[] capabilities
string mission_status
string[] discovered_targets
string communication_data
